adapters/krushmedia/krushmedia_test.go

```diff
@@ -0,0 +1,11 @@
+package krushmedia
+
+import (
+	"testing"
+
+	"github.com/prebid/prebid-server/adapters/adapterstest"
+)
+
+func TestJsonSamples(t *testing.T) {
+	adapterstest.RunJSONBidderTest(t, "krushmediatest", NewKrushmediaBidder("http://example.com/?c=rtb&m=req&key=2"))
```

Comment:
You need to have a macro in here for `AccountID` otherwise it will never be able to take the account ID in the bidder ext and replace it in the URL for the tests. The `RunJSONBidderTest` basically runs all the JSON tests you have defined below.